test/integration/messaging.js

```diff
@@ -27,25 +29,29 @@ describe('messaging', function() {
   });
 
   it('updates a message', function(done) {
-    senderModel.update({topic: 'public', message: 'An update message'},
-    function(err) {
-      if (err) return done(err);
-      receiverModel.find(0, function(msg) {
-        expect(msg).to.equal('An update message');
-        done();
-      });
-    });
+    setTimeout(function() {
+      senderModel.update({topic: 'public', message: 'An update message'},
+        function(err) {
+          if (err) return done(err);
+          receiverModel.find(0, function(msg) {
+            expect(msg).to.equal('An update message');
+            done();
+          });
+        });
+    }, timeout);
   });
 
   it('deletes a message', function(done) {
-    senderModel.delete({topic: 'public', message: 'A delete message'},
-    function(err) {
-      if (err) return done(err);
-      receiverModel.find(0, function(msg) {
-        expect(msg).to.equal('A delete message');
-        done();
-      });
-    });
+    setTimeout(function() {
+      senderModel.delete({topic: 'public', message: 'A delete message'},
+        function(err) {
+          if (err) return done(err);
+          receiverModel.find(0, function(msg) {
+            expect(msg).to.equal('A delete message');
+            done();
+          });
+        });
+    }, timeout);
```

Comment:
I wouldn't make these tests interdependent; it makes them unnecessarily fragile.
The `updates a message` test should make its own message, and clean up its own messages.
The `deletes a message` test should create its own message to delete.

As it's currently written, a failure in the create would cascade through to the remaining items.